src/main/java/me/mrCookieSlime/Slimefun/Events/DisenchanterEvent.java

```diff
@@ -0,0 +1,42 @@
+package me.mrCookieSlime.Slimefun.Events;
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+public class DisenchanterEvent extends Event implements Cancellable {
+	
+	private static final HandlerList handlers = new HandlerList();
+
+	private ItemStack sfItem;
+	private boolean cancelled;
+
+	public HandlerList getHandlers() {
+	    return handlers;
+	}
+	
+	public static HandlerList getHandlerList() {
+        return handlers;
+    }
+	
+	public DisenchanterEvent(ItemStack sfitem){
+		super(true);
```

Comment:
Is this async? I don't think it is so why specify it as so?

Reply:
It's async, it have to bee async.